### DISCUSSION TITLE
DeepMD simulate monolayer adsorption

### DISCUSSION THREAD

**Discussion by nsysu7433111** [deepmodeling/deepmd-kit#4863] [General]
Dear developers and team,
Thanks to the developers and development team of DeePMD for creating a very accurate potential model, which has allowed us to take a huge step forward in our progress on the atomic scale.

I have a question about how to set up a DeepMD simulation for the adsorption of a 2D material monolayer. 
Our simulation workflow is as follows: after the DLP training process is complete, we use DLP to perform a minimization (Figure 1). We then use the final minimized structure to perform an NPT simulation (Figures 2 and 3).
However, when I use DP-Lammps to run the NPT simulation for the monolayer adsorption, the structure becomes unstable (it doesn't hold the whole surface), as shown in Figure 3.

My training set :
1.Substrate 50,300K (BC2N,BC2As,BC2P)
2.Metal Lithium 300K (Adsorption Atom)
3.Dimer (Contain Substrate and Adsorption Atom)
4.Full of Li Adsorption on Substrate(One&Both side )
5.Two of Li adsorption on Substrate(Only one side)

Descriptor Settings
 Descriptor: hybrid
  type: se_e2_a
  rcut: 6.26130000000001
  rcut_smth: 2.4455000001
  type: se_e3
  rcut: 4.00000001
  rcut_smth: 2.00000001

Loss Function Weights
  start_pref_f : 1000
  limit_pref_f : 1
  start_pref_v : 100
  limit_pref_v : 0.1
  start_pref_e : 0.001
  limit_pref_e : 1

<img width="830" height="558" alt="Min(Front View)" src="https://github.com/user-attachments/assets/e96f50bd-e927-4e22-b144-ba179279ff59" />

Fig.1 Minimize final data 

<img width="1158" height="809" alt="NPT(before crash)" src="https://github.com/user-attachments/assets/a21155f3-6daf-41c4-a7cb-355b35157a40" />
Fig.2 Npt simulation(Before Crash)

<img width="1235" height="795" alt="NPT(crash)" src="https://github.com/user-attachments/assets/081046c2-16b7-483c-9d45-790ac1b3bec1" />
Fig.3 Npt simulation(Crash)

**Comment by wanghan-iapcm** [comment]
please refer to https://github.com/deepmodeling/deepmd-kit/discussions/4802#discussioncomment-13491850

**Reply by nsysu7433111** [reply]
Thank you, Dr. Wang Han, for your reply. I will proceed with the work based on your suggestions.
